packages/hash/api/src/graphql/typeDefs/paginationConnections.typedef.ts

```diff
@@ -0,0 +1,66 @@
+import { gql } from "apollo-server-express";
+
+export const pagePaginationTypedef = gql`
+  """
+  Each pagination specifies if there are more results to request.
```

Comment:
We would also ideally describe the purpose/summary of each type - once we have hash.dev up we can generate docs for the API directly from these files to show there.
```suggestion
  Information returned with paginated queries.
  
  Each pagination specifies if there are more results to request.
```